exceptions.cabal

```diff
@@ -53,6 +57,14 @@ library
   if !impl(ghc >= 8.0)
```

Comment:
Don't you need to remove the `transformers-compat` dependency [above](https://github.com/ekmett/exceptions/pull/74/files#diff-451f08946ce59d8d252355dc3fc83332R54)?

Reply:
Oops. Yes, good catch.